package.json

```diff
@@ -67,6 +67,7 @@
     "@babel/preset-stage-3": "^7.0.0-beta.35",
     "@babel/preset-typescript": "^7.0.0-beta.35",
     "@babel/register": "^7.0.0-beta.35",
+    "@babel/runtime": "^7.0.0-beta.35",
```

Comment:
What's the reason for moving this from **normal** to **dev** dependency?

The `@babel/runtime` is needed to have runtime code working, so it should be either in the `dependencies`, OR in both `devDependencies` and `peerDependencies` to make sure it's installed in the parent project.

Reply:
Ok, I'll move this back